Source/Dafny/DafnyOptions.cs

```diff
@@ -980,6 +980,10 @@ not sound
         /verificationLogger:trx;LogFileName=<...>.
     The exact mapping of verification concepts to the TRX and CSV formats is
     experimental and subject to change!
+
+    The CSV format also accepts a `Metric` parameter to indicate which metric
```

Comment:
Is there a good reason we couldn't just emit both as separate columns all the time, for simplicity?

Reply:
The key reason for this is to prepare for including multiple samples for each proof obligation, along with statistics about those (variance, and so on). With a single metric, those statistics will always be in the same fields and can be processed uniformly.

Reply:
The latest commit includes all metrics, with the idea that there could be multiple rows in the output for a given proof obligation and that external tools could compute any statistics of interest.